UKHomeOffice/sgmr-service
# Pass voyageId on cancel click

### AC / Description of changes

Update the 'cancel' button click to pass the voyageId so it doesn't get lost from state


### To Test

- CYA page
- Click change on the "people on board"
- Click "save changes" ( leave it with the one person I had already selected / have saved in my acct)
- Get taken to the people-manifest page
- Click "add new person"
- Click "exit without saving"
- Click "cancel changes"
- > you should be returned to the CYA page


### Notes

---

**Remember to merge to feature branches not `master`.**
